emission/storage/decorations/tour_model_queries.py

```diff
@@ -33,17 +33,18 @@ def get_tour_model(user_id):
 def get_common_places(user_id):
     db = edb.get_common_place_db()
     place_jsons = db.find({"user_id" : user_id})
-    return [esdcpq.make_common_place_from_json(place_json) for place_json in place_jsons]
+    return [esdcpq.make_common_place(place_json) for place_json in place_jsons]
```

Comment:
I'm going to merge this now, but I'd appreciate a test for this before you move on to other things. Also, on Friday, maybe you can walk me through the tests that you have...